components/library/qt6/Makefile

```diff
@@ -0,0 +1,150 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Daniel Chan
+#
+
+BUILD_BITS=		64_and_32
+
```

Comment:
Do we really want 32-bit? This is a new package and we don't have any consumers yet. We want to drop 32-bit where possible.

Reply:
Yes, we can drop it since no user for 32bit qt6 yet. And this can reduce the building time.